Quality.md

```diff
@@ -0,0 +1,31 @@
+## Quality Assistance
```

Comment:
I like how you have moved this outside the engineering section which was getting crowded.

Consider linking to the Quality Assistance document: https://octopushq.atlassian.net/wiki/spaces/IN/pages/892370955/Quality+Assistance